src/vivarium_gates_iv_iron/constants/results.py

```diff
@@ -84,9 +84,9 @@
 
 POP_STATES = ("living", "dead", "tracked", "untracked")
 SEXES = ("female",)
-YEARS = tuple(range(2024, 2041))
+YEARS = tuple(range(2019, 2033))
```

Comment:
Did you mean to change this date range?

Reply:
These are the years specific to the stillbirth v&v run.